adm/style/acp_phpbb_ideas.html

```diff
@@ -28,16 +28,17 @@
 			<dd>{options.CONTENT}</dd>
 			<!-- IF options.KEY == 'ideas_poster_id' --><dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd><!-- ENDIF -->
 		</dl>
+		<!-- IF options.KEY == 'ideas_base_url' -->
```

Comment:
We use its methods as well as validation actually https://github.com/phpbb/phpbb-ideas/pull/39/files#diff-cd1c3b451ea8b3050b573a187c85a2d8L115
But if you think we should rewrite the module, I'm ok with that.